### DISCUSSION TITLE
zsh: segmentation fault  ruff

### DISCUSSION THREAD

**Discussion by isJokers** [astral-sh/ruff#9824] [Q&A] [answered]
I updated ruff to 0.2.0. Then I get the following error when I use it

```
zsh: segmentation fault  ruff
```

Does anyone know why？ I rolled back to 0.1.9 and it still worked, but 0.2.0 didn't.

**Comment by MichaReiser** [comment]
Hy @isJokers 

Can you tell us more about how you run ruff? I ran ruff in a zsh terminal and I wasn't able to reproduce the segfault but I expect that's because I run the wrong command.

**Reply by MichaReiser** [reply, marked as answer]
This is probably related to https://github.com/astral-sh/ruff/issues/9823

**Comment by charliermarsh** [comment]
I expect this to be fixed by https://github.com/astral-sh/ruff/pull/9834.
